examples/psp1155extensions/psp1155mintable/.gitignore

```diff
@@ -0,0 +1,9 @@
+# Ignore build artifacts from the local tests sub-crate.
```

Comment:
You don't need to duplicate `psp1155`, because you are in the folder for `psp1155`)
Lets use next naming `psp1155_extensions/mintable`. Do the same for other extensions